dashboard/v1.1/src/pages/Input/Input.tsx

```diff
@@ -82,6 +132,80 @@ const Input: FC<{}> = () => {
     setApplyParams(false);
     setApplyBody(false);
   };
+  const testAndEdit = () => {
+    const params = {};
```

Comment:
Can the length of this function be reduced? If not, please consider having a different file for `testAndEdit` and `testURL` to have this file short.